apis/config/v1beta1/configuration_types.go

```diff
@@ -472,3 +476,17 @@ type FairSharing struct {
 	// The default strategy is ["LessThanOrEqualToFinalShare", "LessThanInitialShare"].
 	PreemptionStrategies []PreemptionStrategy `json:"preemptionStrategies,omitempty"`
 }
+
+type AdmissionFairSharing struct {
+	// usageHalfLifeTime indicates the time after which the current usage will decay by a half
+	// If set to 0, usage will be reset to 0.
```

Comment:
Do you mean it is reset to 0 immediately? 

0 is the default? Also, do we have use-cases for 0?

Reply:
> Do you mean it is reset to 0 immediately?

Exactly

> 0 is the default?

No, we don't have a default for HalfTime

> Also, do we have use-cases for 0?

Yes, we would like to allow user to quickly reset values in FairSharingStatus if they're not happy with the current state

Reply:
> Yes, we would like to allow user to quickly reset values in FairSharingStatus if they're not happy with the current state

I don't understand this use case actually, because if they set to 0, then it will be reset all the time, without user intervention, so I'm wondering how this can be useful.